src/style.css

```diff
@@ -1 +1,80 @@
-/* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 116px;
+  align-items: flex-start;
+}
+
+.stars {
+  display: flex;
+  margin-right: 4px;
```

Comment:
no need to use margin-right here, each star should have it, but container for them shouldn't